### PR TITLE
chore:  add rust husky

### PR DESCRIPTION
**Pull request by MatejVukosav**
- use husky-rs instead of regular husky
- use dprint instead of prettier
- fix md files based on dprint
- remove unused eslint files

> Note
> After this is merged; execute `cargo clean `and `cargo build` to setup husky

### CODE REVIEW

File: crates/merod/Cargo.toml [hunk 1]
```diff
@@ -36,6 +36,7 @@ calimero-node = { path = "../node" }
 calimero-network = { path = "../network" }
 calimero-server = { path = "../server", features = ["jsonrpc", "websocket", "admin"] }
 calimero-store = { path = "../store" }
+husky-rs.workspace = true # activate husky hooks
```
Comment (miraclx) [inline crates/merod/Cargo.toml:39]: shouldn't it be `[dev-dependencies]`?

File: .prettierignore
Comment (miraclx) [inline .prettierignore:1]: are these accounted for someway with dprint?

File: Cargo.lock
Comment (miraclx) [inline Cargo.lock:1]: a lot more changed here than needed to, try this

```console
$ git checkout master Cargo.lock
```

to restore the lockfile version from master, then `cargo build` on top of that

Wholesale dependency upgrades which `cargo update` does must be handled more consciously, ideally a separate effort.

Comment (miraclx) [reply Cargo.lock:1]: restore & build also seems to do a wholesale update in this case, this appears to be the minimum lockfile requirement 👍🏽

Comment (MatejVukosav) [reply Cargo.lock:1]: yes, it works,  just pushed. tnx!

**Comment by MatejVukosav** [comment]
Bump

**Pull request closed without merging**